okio/src/commonTest/kotlin/okio/AbstractBufferedSourceTest.kt

```diff
@@ -1013,6 +1013,7 @@ abstract class AbstractBufferedSourceTest internal constructor(
     assertTrue(source.exhausted())
   }
 
+  /** Note that this test crashes the VM on Android. */
```

Comment:
Planning to address this or ignore the test in a followup?

Reply:
Hoping to fix in follow-up!